### DISCUSSION TITLE
is Budgie still switching to EFL?

### DISCUSSION THREAD

**Discussion by daemonspudguy** [BuddiesOfBudgie/budgie-desktop#533] [General]
The last time this was mentioned was 2022, so I am curious if Budgie is indeed still switching to EFL.

**Comment by JoshStrobl** [comment]
The details of any technical choices for Budgie 11 under Buddies of Budgie will be done through our blog. At this point, myself and the rest of the team would rather just _show_ people Budgie 11 than describe Budgie and prescribing its use of various technologies, of which will have no meaningful impact to most end users.
